server/utils/oauth2connect/oauth2_complete.go

```diff
@@ -4,12 +4,13 @@
 package oauth2connect
 
 import (
+	"fmt"
 	"net/http"
 
 	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/httputils"
 )
 
-func (oa *oa) oauth2Complete(w http.ResponseWriter, r *http.Request) {
+func (oa *oa) oauth2Complete(w http.ResponseWriter, r *http.Request, providerDisplayName string) {
```

Comment:
`oa` already has access to `config.ProviderConfig`, so I think we don't need the `providerDisplayName` parameter right?

Reply:
Oh right, that was from a previous change.